README.md

```diff
@@ -3,88 +3,104 @@
 
-```shell
-git clone https://github.com/mindee/doctr.git
-pip install -e doctr/.
+model = ocr_predictor(det_arch='db_resnet50', reco_arch='crnn_vgg16_bn', pretrained=True)
 ```
 
+### Reading files
 
-## Usage
-
-### Python package
+Documents can be interpreted from PDF or images:
 
-The python package provides you with tools to read files and detect all text information they are holding.
+```python
+from doctr.documents import DocumentFile
+# PDF
+pdf_doc = DocumentFile.from_pdf("path/to/your/doc.pdf")
+# Image
+single_img_doc = DocumentFile.from_images("path/to/your/img.jpg")
+# Multiple page images
+multi_img_doc = DocumentFile.from_images(["path/to/page1.jpg", "path/to/page2.jpg"])
+```
 
+### Putting it together
+Let's use the default pretrained model for an example:
 ```python
 from doctr.documents import DocumentFile
 from doctr.models import ocr_predictor
 
 model = ocr_predictor(pretrained=True)
 # PDF
 doc = DocumentFile.from_pdf("path/to/your/doc.pdf")
-# Image
-# doc = DocumentFile.from_images("path/to/your/img.jpg")
-# Multiple page images
-# doc = DocumentFile.from_images(["path/to/page1.jpg", "path/to/page2.jpg"])
 # Analyze
 result = model(doc)
-# Export
-json_output = result.export()
-# Visualize
+```
+
+To make sense of your model's predictions, you can visualize them as follows:
+
+```python
 result.show(doc)
 ```
 
 ![DocTR example](https://github.com/mindee/doctr/releases/download/v0.1.1/doctr_example_script.gif)
 
-For an exhaustive list of pretrained models available, please refer to the [documentation](https://mindee.github.io/doctr/models.html).
+or export them to JSON format (to get a better understanding of our document model, check our [documentation](https://mindee.github.io/doctr/documents.html#document-structure)):
 
-### Example script
+```python
+json_output = result.export()
+```
 
-![DocTR example](https://github.com/mindee/doctr/releases/download/v0.1.1/doctr_example_script.gif)
 
-An example script is provided for a simple documentation analysis of a PDF file:
+## Installation
+
+Python 3.6 (or higher) and [pip](https://pip.pypa.io/en/stable/) are required to install DocTR.
+
+You can install the latest release of the package using [pypi](https://pypi.org/project/python-doctr/) as follows:
 
 ```shell
-python scripts/analyze.py path/to/your/doc.pdf
+pip install python-doctr
+```
+
+Or you can install it from source:
+
+```shell
+git clone https://github.com/mindee/doctr.git
+pip install -e doctr/.
 ```
-All script arguments can be checked using `python scripts/analyze.py --help`
+
+
+## Models architectures
+Credits where it's due: this repository is implementing, among others, architectures from published research papers.
+
+### Text Detection
+- [Real-time Scene Text Detection with Differentiable Binarization](https://arxiv.org/pdf/1911.08947.pdf).
+
+### Text Recognition
+- [Convolutional RNN: an Enhanced Model for Extracting Features from Sequential Data](https://arxiv.org/pdf/1602.05875.pdf).
```

Comment:
The paper for the CRNN is the following: https://arxiv.org/abs/1507.05717 (An End-to-End Trainable Neural Network for Image-based Sequence Recognition and Its Application to Scene Text Recognition)

Reply:
Good to know, I just fixed the other reference to the paper!